### DISCUSSION TITLE
Option to sort buffers according to "time last used"

### DISCUSSION THREAD

**Discussion by ChHaeni** [nvim-lualine/lualine.nvim#717] [Ideas]
I find it quite usefull when the order of `buffers` is displayed according to when the buffers were last active (as with `:ls t`): [see here](https://github.com/nvim-lualine/lualine.nvim/compare/master...ChHaeni:sort_buffers)

Would this be worth a PR, including it as a `buffers` option (e.g. `sort_last_accessed=true`)?
Are there any possible pitfalls?

However, I'm rather inexperienced with lua and I'm unsure how to write my code better/more efficient...

**Comment by shadmansaleh** [comment]
Open a pr . I'll look into the code later.

**Reply by ChHaeni** [reply]
Ok done #729
